### PR TITLE
Add prepend operation for custom munge

### PR DESCRIPTION
**Pull request by LuomaJuha**
Noticed that there is an append operation so decided to add prepend also.

### CODE REVIEW

File: module/VuFind/tests/fixtures/configs/1.1/searchspecs.yaml [hunk 1]
```diff
@@ -120,6 +120,7 @@
 # [lowercase] - Convert string to lowercase
 # [preg_replace, pattern, replacement] - Perform a regular expression replace
 #                                        using the preg_replace() PHP function
+# [prepend, text] - Prepend text to the beginning of the user's search string
```
Comment (demiankatz) [inline module/VuFind/tests/fixtures/configs/1.1/searchspecs.yaml]: You don't need to add this to the legacy-version-specific test fixtures -- these are historical versions of the configuration files used to test upgrade/migration, so they should not be changed.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks. @LuomaJuha, this makes sense to me -- just one minor request regarding the 1.x test fixtures.

File: module/VuFind/tests/fixtures/configs/1.1/searchspecs.yaml (hunk 1 above)
Comment (LuomaJuha) [reply module/VuFind/tests/fixtures/configs/1.1/searchspecs.yaml]: oh, okay! will remove it from them

**Review by demiankatz — APPROVED** [review]
Thanks, @LuomaJuha!

**Pull request merged**